src/rabbit_policy.erl

```diff
@@ -242,6 +242,8 @@ update_policies(VHost) ->
                  fun() ->
                          [mnesia:lock({table, T}, write) || T <- Tabs], %% [1]
                          case catch list(VHost) of
+                             {'EXIT', Exit} ->
+                                 exit(Exit);
```

Comment:
It should be `exit()` because:
* If we caught an exit signal triggered by `mnesia:abort()`, the tuple is already of the form expected by Mnesia (`{aborted, Some_Internal_Record}`). So here, we just forward it.
* If we caught a "normal" exit signa, we also forward it as is and let Mnesia handle the difference between an abortion and a plain exit signal.